operator/integration-tests/model-in-image/mii-sample-wrapper/mii-domain.yaml.template-JRF-AI

```diff
@@ -167,3 +150,18 @@ spec:
 
       # Name of secret with walletFile containing base64 encoded opss wallet, used for JRF domains
       #walletFileSecret: @@DOMAIN_UID@@-opss-walletfile-secret
+
+---
+
+apiVersion: "weblogic.oracle/v1"
+kind: Cluster
+metadata:
+  name: cluster-1
```

Comment:
The name should be decorated with the DOMAIN_UID ( @@DOMAIN_UID@@-cluster-1 ). The Mii sample has a step that concurrently runs two domains.

Reply:
(We should use a similar pattern for _all_ samples that define a Cluster - not just MII.)